### PR TITLE
Feat(web): Introduce `Dropdown` component (refs #DS-326)

### PR DESCRIPTION
**Pull request by vmilersky**

### CODE REVIEW

File: packages/web/src/scss/components/Dropdown/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{{#> layout/plain }}
+
+<div class="Dropdown is-open">
+  Content
```
Comment (literat) [inline packages/web/src/scss/components/Dropdown/index.html:4]: I would prefer to add more content. Now it looks like some kind of weird button in the demo page.

**Review by literat — CHANGES_REQUESTED** [review]

File: packages/web/src/scss/components/Dropdown/index.html (hunk 1 above)
Comment (vmilersky) [reply packages/web/src/scss/components/Dropdown/index.html]: Link items added.

**Review by literat — APPROVED** [review]
LGTM

File: packages/web/src/scss/components/Dropdown/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Dropdown
+
+```html
+<div class="Dropdown is-open">
+  <a href="#" class="d-flex mb-400">
+    <svg width="24" height="24" aria-hidden="true" class="mr-400">
+      <use xlink:href="/icons/svg/sprite.svg#info"></use>
```
Comment (crishpeen) [inline packages/web/src/scss/components/Dropdown/README.md:7]: self closing please :) 
```suggestion
      <use xlink:href="/icons/svg/sprite.svg#info" />
```

File: packages/web/src/scss/components/Dropdown/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@use '@tokens' as tokens;
+
+$padding: tokens.$space-600;
+$border-radius: tokens.$radius-200;
+$shadow: tokens.$shadow-100;
+$background: tokens.$background-basic;
```
Comment (crishpeen) [inline packages/web/src/scss/components/Dropdown/_theme.scss:6]: can we sort this the same way as in the class directly?

1. padding
2. background
3. radius
4. shadow

**Review by crishpeen — APPROVED** [review]

File: packages/web/src/scss/components/Dropdown/README.md (hunk 2 above)
Comment (vmilersky) [reply packages/web/src/scss/components/Dropdown/README.md]: Fixed.

**Pull request merged**